rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -2351,6 +2216,43 @@ public JobIdData submit(@HeaderParam("sessionid") String sessionId, @PathParam("
         }
     }
 
+    @Override
+    public JobIdData reSubmit(String sessionId, String jobId, PathSegment pathSegment, UriInfo contextInfos)
+            throws JobCreationRestException, NotConnectedRestException, PermissionRestException,
+            SubmissionClosedRestException, IOException {
+        Scheduler scheduler = checkAccess(sessionId, "reSubmit");
+
+        File tmpJobFile = File.createTempFile("job", "d");
+
+        JobId newJobId;
+
+        try {
+            final String jobContent = scheduler.getJobContent(JobIdImpl.makeJobId(jobId));
+            FileUtils.write(tmpJobFile, jobContent, Charset.forName("UTF-8"));
+        } catch (NotConnectedException e) {
+            e.printStackTrace();
+        } catch (UnknownJobException e) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Combine this catch with the one at line 2232, which has the same body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2147)